Fixed spam caused by etcdserver/api/v2http logging through capnslog, …

aaron-prindle opened the pull request:
…changed etcdserver/api logging to be for NOTICE lvl and above

#223

dlorenc commented on cmd/localkube/cmd/start.go:
```diff
@@ -37,6 +38,9 @@ func StartLocalkube() {
 		os.Exit(0)
 	}
 
+	apiRepoLogger, _ := capnslog.GetRepoLogger("github.com/coreos/etcd/etcdserver/api")
```
Could you add a comment here explaining why we're doing this?

dlorenc commented:
Nice, one nit then LGTM. Could you attach the output of minikube logs here as well?

aaron-prindle commented:
Here is the log output now with the changes made:
==> /var/log/localkube.err <==
Starting etcd...
Starting apiserver...
Starting controller-manager...
Starting scheduler...
Starting kubelet...
Starting proxy...
Starting dns...

==> /var/log/localkube.out <==
I0630 17:32:25.791361    2996 server.go:201] Using iptables Proxier.
I0630 17:32:25.791417    2996 server.go:214] Tearing down userspace rules.
2016-06-30 17:32:25.799285 I | etcdserver: name = kubeetcd
2016-06-30 17:32:25.799296 I | etcdserver: data dir = /var/lib/localkube/etcd
2016-06-30 17:32:25.799300 I | etcdserver: member dir = /var/lib/localkube/etcd/member
2016-06-30 17:32:25.799304 I | etcdserver: heartbeat = 100ms
2016-06-30 17:32:25.799309 I | etcdserver: election = 1000ms
2016-06-30 17:32:25.799312 I | etcdserver: snapshot count = 10000
2016-06-30 17:32:25.799318 I | etcdserver: advertise client URLs = http://localhost:2379
2016-06-30 17:32:25.799321 I | etcdserver: initial advertise peer URLs = http://localhost:2380
2016-06-30 17:32:25.799329 I | etcdserver: initial cluster = kubeetcd=http://localhost:2380
2016-06-30 17:32:25.820103 I | etcdserver: starting member 37807cb0bf7500f6 in cluster 2c833ae9c7555b5e
2016-06-30 17:32:25.820142 I | raft: 37807cb0bf7500f6 became follower at term 0
2016-06-30 17:32:25.820156 I | raft: newRaft 37807cb0bf7500f6 [peers: [], term: 0, commit: 0, applied: 0, lastindex: 0, lastterm: 0]
2016-06-30 17:32:25.820164 I | raft: 37807cb0bf7500f6 became follower at term 1
2016-06-30 17:32:25.820663 I | etcdserver: starting server... [version: 2.3.0+git, cluster version: to_be_decided]
2016-06-30 17:32:25.821028 I | etcdserver: name = dns
2016-06-30 17:32:25.821037 I | etcdserver: data dir = /var/lib/localkube/dns
2016-06-30 17:32:25.821043 I | etcdserver: member dir = /var/lib/localkube/dns/member
2016-06-30 17:32:25.821048 I | etcdserver: heartbeat = 100ms
2016-06-30 17:32:25.821053 I | etcdserver: election = 1000ms
2016-06-30 17:32:25.821057 I | etcdserver: snapshot count = 10000
2016-06-30 17:32:25.821065 I | etcdserver: advertise client URLs = http://localhost:9090
2016-06-30 17:32:25.821072 I | etcdserver: initial advertise peer URLs = http://localhost:9256
2016-06-30 17:32:25.821081 I | etcdserver: initial cluster = dns=http://localhost:9256
I0630 17:32:25.822198    2996 conntrack.go:36] Setting nf_conntrack_max to 262144
I0630 17:32:25.822243    2996 conntrack.go:41] Setting conntrack hashsize to 65536
I0630 17:32:25.822415    2996 conntrack.go:46] Setting nf_conntrack_tcp_timeout_established to 86400
I0630 17:32:25.832416    2996 genericapiserver.go:606] Will report 10.0.2.15 as public IP address.
I0630 17:32:25.833814    2996 genericapiserver.go:288] Node port range unspecified. Defaulting to 30000-32767.
E0630 17:32:25.898865    2996 controllermanager.go:122] unable to register configz: register config "componentconfig" twice
I0630 17:32:25.899506    2996 plugins.go:71] No cloud provider specified.
W0630 17:32:25.899513    2996 controllermanager.go:235] Unsuccessful parsing of cluster CIDR : invalid CIDR address: 
W0630 17:32:25.899519    2996 controllermanager.go:239] Unsuccessful parsing of service CIDR : invalid CIDR address: 
I0630 17:32:25.899574    2996 nodecontroller.go:167] Sending events to api server.
E0630 17:32:25.899741    2996 controllermanager.go:250] Failed to start service controller: ServiceController should not be run without a cloudprovider.
W0630 17:32:25.899754    2996 controllermanager.go:265] allocate-node-cidrs set to false, will not configure cloud provider routes.
E0630 17:32:25.899854    2996 util.go:45] Metric for replenishment_controller already registered
E0630 17:32:25.899862    2996 util.go:45] Metric for replenishment_controller already registered
E0630 17:32:25.899867    2996 util.go:45] Metric for replenishment_controller already registered
E0630 17:32:25.899872    2996 util.go:45] Metric for replenishment_controller already registered
E0630 17:32:25.899877    2996 util.go:45] Metric for replenishment_controller already registered
E0630 17:32:25.899987    2996 server.go:319] unable to register configz: register config "componentconfig" twice
E0630 17:32:25.925043    2996 reflector.go:205] pkg/proxy/config/api.go:33: Failed to list *api.Endpoints: Get http://127.0.0.1:8080/api/v1/endpoints?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.925099    2996 reflector.go:205] pkg/proxy/config/api.go:30: Failed to list *api.Service: Get http://127.0.0.1:8080/api/v1/services?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.925125    2996 server.go:75] unable to register configz: register config "componentconfig" twice
I0630 17:32:25.925806    2996 replication_controller.go:240] Starting RC Manager
2016-06-30 17:32:25.926784 I | etcdserver: starting member 6429d4ab4d8a33da in cluster d3c354cfe77b0b70
2016-06-30 17:32:25.926808 I | raft: 6429d4ab4d8a33da became follower at term 0
2016-06-30 17:32:25.926820 I | raft: newRaft 6429d4ab4d8a33da [peers: [], term: 0, commit: 0, applied: 0, lastindex: 0, lastterm: 0]
2016-06-30 17:32:25.926824 I | raft: 6429d4ab4d8a33da became follower at term 1
2016-06-30 17:32:25.927053 I | etcdserver: starting server... [version: 2.3.0+git, cluster version: to_be_decided]
2016-06-30 17:32:25.927758 N | etcdserver: added local member 37807cb0bf7500f6 [http://localhost:2380] to cluster 2c833ae9c7555b5e
2016-06-30 17:32:25.951271 I | skydns: ready for queries on cluster.local. for tcp://10.0.2.15:53 [rcache 0]
2016-06-30 17:32:25.951302 I | skydns: ready for queries on cluster.local. for udp://10.0.2.15:53 [rcache 0]
E0630 17:32:25.971906    2996 controllermanager.go:298] Failed to get api versions from server: Get http://127.0.0.1:8080/api: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988147    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/factory/factory.go:387: Failed to list *api.Pod: Get http://127.0.0.1:8080/api/v1/pods?fieldSelector=spec.nodeName%3D%2Cstatus.phase%21%3DFailed%2Cstatus.phase%21%3DSucceeded&resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988210    2996 nodecontroller.go:293] Error monitoring node status: Get http://127.0.0.1:8080/api/v1/nodes: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988258    2996 reflector.go:216] pkg/controller/node/nodecontroller.go:288: Failed to list *extensions.DaemonSet: Get http://127.0.0.1:8080/apis/extensions/v1beta1/daemonsets?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988306    2996 reflector.go:216] pkg/controller/node/nodecontroller.go:287: Failed to list *api.Pod: Get http://127.0.0.1:8080/api/v1/pods?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988343    2996 reflector.go:216] pkg/controller/node/nodecontroller.go:286: Failed to list *api.Node: Get http://127.0.0.1:8080/api/v1/nodes?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988598    2996 reflector.go:216] pkg/controller/gc/gc_controller.go:89: Failed to list *api.Pod: Get http://127.0.0.1:8080/api/v1/pods?fieldSelector=status.phase%21%3DPending%2Cstatus.phase%21%3DRunning%2Cstatus.phase%21%3DUnknown&resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988863    2996 reflector.go:205] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/admission/resourcequota/controller.go:121: Failed to list *api.ResourceQuota: Get http://127.0.0.1:8080/api/v1/resourcequotas?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988909    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/admission/serviceaccount/admission.go:119: Failed to list *api.Secret: Get http://127.0.0.1:8080/api/v1/secrets?fieldSelector=type%3Dkubernetes.io%2Fservice-account-token&resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.988965    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/admission/serviceaccount/admission.go:103: Failed to list *api.ServiceAccount: Get http://127.0.0.1:8080/api/v1/serviceaccounts?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.989003    2996 reflector.go:205] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/admission/limitranger/admission.go:154: Failed to list *api.LimitRange: Get http://127.0.0.1:8080/api/v1/limitranges?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.989049    2996 reflector.go:205] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/admission/namespace/lifecycle/admission.go:116: Failed to list *api.Namespace: Get http://127.0.0.1:8080/api/v1/namespaces?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.989090    2996 event.go:207] Unable to write event: 'Post http://127.0.0.1:8080/api/v1/namespaces/default/events: dial tcp 127.0.0.1:8080: getsockopt: connection refused' (may retry after sleeping)
E0630 17:32:25.989746    2996 reflector.go:216] pkg/controller/replication/replication_controller.go:241: Failed to list *api.ReplicationController: Get http://127.0.0.1:8080/api/v1/replicationcontrollers?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.990974    2996 reflector.go:216] pkg/controller/endpoint/endpoints_controller.go:157: Failed to list *api.Service: Get http://127.0.0.1:8080/api/v1/services?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
2016-06-30 17:32:25.991300 N | etcdserver: added local member 6429d4ab4d8a33da [http://localhost:9256] to cluster d3c354cfe77b0b70
E0630 17:32:25.991365    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/factory/factory.go:413: Failed to list *extensions.ReplicaSet: Get http://127.0.0.1:8080/apis/extensions/v1beta1/replicasets?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.991398    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/factory/factory.go:408: Failed to list *api.ReplicationController: Get http://127.0.0.1:8080/api/v1/replicationcontrollers?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.991424    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/factory/factory.go:403: Failed to list *api.Service: Get http://127.0.0.1:8080/api/v1/services?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.991472    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/factory/factory.go:398: Failed to list *api.PersistentVolumeClaim: Get http://127.0.0.1:8080/api/v1/persistentvolumeclaims?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.991618    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/factory/factory.go:397: Failed to list *api.PersistentVolume: Get http://127.0.0.1:8080/api/v1/persistentvolumes?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.991647    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/factory/factory.go:393: Failed to list *api.Node: Get http://127.0.0.1:8080/api/v1/nodes?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.991676    2996 reflector.go:216] k8s.io/minikube/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/factory/factory.go:390: Failed to list *api.Pod: Get http://127.0.0.1:8080/api/v1/pods?fieldSelector=spec.nodeName%21%3D%2Cstatus.phase%21%3DFailed%2Cstatus.phase%21%3DSucceeded&resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.991707    2996 reflector.go:216] pkg/controller/resourcequota/resource_quota_controller.go:195: Failed to list *api.Service: Get http://127.0.0.1:8080/api/v1/services?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:25.991732    2996 reflector.go:216] pkg/controller/resourcequota/resource_quota_controller.go:192: Failed to list *api.ResourceQuota: Get http://127.0.0.1:8080/api/v1/resourcequotas?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
I0630 17:32:25.991774    2996 dns.go:156] Failed to check for kube-system namespace existence: Get http://localhost:8080/api/v1/namespaces/kube-system: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:26.009954    2996 reflector.go:216] pkg/controller/resourcequota/resource_quota_controller.go:195: Failed to list *api.ConfigMap: Get http://127.0.0.1:8080/api/v1/configmaps?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:26.010184    2996 reflector.go:216] pkg/controller/resourcequota/resource_quota_controller.go:195: Failed to list *api.Secret: Get http://127.0.0.1:8080/api/v1/secrets?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:26.010219    2996 reflector.go:216] pkg/controller/resourcequota/resource_quota_controller.go:195: Failed to list *api.PersistentVolumeClaim: Get http://127.0.0.1:8080/api/v1/persistentvolumeclaims?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:26.010246    2996 reflector.go:216] pkg/controller/resourcequota/resource_quota_controller.go:195: Failed to list *api.ReplicationController: Get http://127.0.0.1:8080/api/v1/replicationcontrollers?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
I0630 17:32:26.070021    2996 kube2sky.go:486] Etcd server found: http://localhost:9090
I0630 17:32:26.160636    2996 docker.go:327] Start docker client with request timeout=2m0s
W0630 17:32:26.162505    2996 server.go:483] Could not load kubeconfig file /var/lib/kubelet/kubeconfig: stat /var/lib/kubelet/kubeconfig: no such file or directory. Trying auth path instead.
W0630 17:32:26.162525    2996 server.go:444] Could not load kubernetes auth path /var/lib/kubelet/kubernetes_auth: stat /var/lib/kubelet/kubernetes_auth: no such file or directory. Continuing with defaults.
I0630 17:32:26.162658    2996 plugins.go:71] No cloud provider specified.
I0630 17:32:26.162709    2996 manager.go:138] cAdvisor running in container: "/"
W0630 17:32:26.167286    2996 manager.go:146] unable to connect to Rkt api service: rkt: cannot tcp Dial rkt api service: dial tcp 127.0.0.1:15441: getsockopt: connection refused
I0630 17:32:26.168347    2996 fs.go:139] Filesystem partitions: map[/dev/sda1:{mountpoint:/mnt/sda1/var/lib/docker/aufs major:8 minor:1 fsType:ext4 blockSize:0}]
I0630 17:32:26.275402    2996 info.go:47] Couldn't collect info from any of the files in "/etc/machine-id,/var/lib/dbus/machine-id"
I0630 17:32:26.275437    2996 manager.go:192] Machine: {NumCores:1 CpuFrequency:2594002 MemoryCapacity:1044250624 MachineID: SystemUUID:1E192134-283E-407D-B961-55A0031A4BB5 BootID:d54c19b5-2daa-4d4d-b24b-2a7a43301a4e Filesystems:[{Device:/dev/sda1 Capacity:19195224064 Type:vfs Inodes:2436448} {Device:tmpfs Capacity:939827200 Type:vfs Inodes:127472}] DiskMap:map[8:0:{Name:sda Major:8 Minor:0 Size:20971520000 Scheduler:deadline} 251:0:{Name:zram0 Major:251 Minor:0 Size:208834560 Scheduler:none}] NetworkDevices:[{Name:dummy0 MacAddress:16:da:e4:2c:1e:cf Speed:0 Mtu:1500} {Name:eth0 MacAddress:08:00:27:1a:72:51 Speed:1000 Mtu:1500} {Name:eth1 MacAddress:08:00:27:75:75:d0 Speed:1000 Mtu:1500}] Topology:[{Id:0 Memory:0 Cores:[{Id:0 Threads:[0] Caches:[{Size:32768 Type:Data Level:1} {Size:32768 Type:Instruction Level:1} {Size:262144 Type:Unified Level:2}]}] Caches:[{Size:4194304 Type:Unified Level:3}]}] CloudProvider:Unknown InstanceType:Unknown InstanceID:None}
I0630 17:32:26.275893    2996 manager.go:198] Version: {KernelVersion:4.4.13-boot2docker ContainerOsVersion:Boot2Docker 1.11.1 (TCL 7.1); master : 0137955 - Sat Jun 18 00:19:25 UTC 2016 DockerVersion:1.11.1 CadvisorVersion: CadvisorRevision:}
I0630 17:32:26.276344    2996 server.go:754] Adding manifest file: /etc/kubernetes/manifests
I0630 17:32:26.276358    2996 server.go:764] Watching apiserver
W0630 17:32:26.276410    2996 kubelet.go:558] Hairpin mode set to "promiscuous-bridge" but configureCBR0 is false, falling back to "hairpin-veth"
I0630 17:32:26.276417    2996 kubelet.go:382] Hairpin mode set to "hairpin-veth"
E0630 17:32:26.279269    2996 reflector.go:205] pkg/kubelet/kubelet.go:280: Failed to list *api.Node: Get http://127.0.0.1:8080/api/v1/nodes?fieldSelector=metadata.name%3D127.0.0.1&resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:26.279309    2996 reflector.go:205] pkg/kubelet/kubelet.go:261: Failed to list *api.Service: Get http://127.0.0.1:8080/api/v1/services?resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
E0630 17:32:26.279342    2996 reflector.go:205] pkg/kubelet/config/apiserver.go:43: Failed to list *api.Pod: Get http://127.0.0.1:8080/api/v1/pods?fieldSelector=spec.nodeName%3D127.0.0.1&resourceVersion=0: dial tcp 127.0.0.1:8080: getsockopt: connection refused
I0630 17:32:26.279501    2996 manager.go:233] Setting dockerRoot to /mnt/sda1/var/lib/docker
I0630 17:32:26.281195    2996 server.go:726] Started kubelet v1.3.0-beta.2
E0630 17:32:26.281827    2996 kubelet.go:929] Image garbage collection failed: unable to find data for container /
I0630 17:32:26.282204    2996 fs_resource_analyzer.go:66] Starting FS ResourceAnalyzer
I0630 17:32:26.282212    2996 manager.go:123] Starting to sync pod status with apiserver
I0630 17:32:26.282226    2996 kubelet.go:2462] Starting kubelet main sync loop.
I0630 17:32:26.282230    2996 kubelet.go:2471] skipping pod synchronization - [network state unknown container runtime is down]
I0630 17:32:26.282418    2996 server.go:117] Starting to listen on 0.0.0.0:10250
I0630 17:32:26.284900    2996 volume_manager.go:192] Starting Kubelet Volume Manager
E0630 17:32:26.286637    2996 event.go:207] Unable to write event: 'Post http://127.0.0.1:8080/api/v1/namespaces/default/events: dial tcp 127.0.0.1:8080: getsockopt: connection refused' (may retry after sleeping)
I0630 17:32:26.287107    2996 factory.go:231] Registering Docker factory
E0630 17:32:26.287122    2996 manager.go:240] Registration of the rkt container factory failed: unable to communicate with Rkt api service: rkt: cannot tcp Dial rkt api service: dial tcp 127.0.0.1:15441: getsockopt: connection refused
I0630 17:32:26.287127    2996 factory.go:54] Registering systemd factory
I0630 17:32:26.287338    2996 factory.go:86] Registering Raw factory
I0630 17:32:26.287554    2996 manager.go:1072] Started watching for new ooms in manager
W0630 17:32:26.287594    2996 manager.go:268] Could not configure a source for OOM detection, disabling OOM events: unable to find any kernel log file available from our set: [/var/log/kern.log /var/log/messages /var/log/syslog]
I0630 17:32:26.287840    2996 manager.go:281] Starting recovery of all containers
I0630 17:32:26.288096    2996 manager.go:286] Recovery completed
2016-06-30 17:32:26.327183 I | raft: 6429d4ab4d8a33da is starting a new election at term 1
2016-06-30 17:32:26.327234 I | raft: 6429d4ab4d8a33da became candidate at term 2
2016-06-30 17:32:26.327239 I | raft: 6429d4ab4d8a33da received vote from 6429d4ab4d8a33da at term 2
2016-06-30 17:32:26.327254 I | raft: 6429d4ab4d8a33da became leader at term 2
2016-06-30 17:32:26.327259 I | raft: raft.node: 6429d4ab4d8a33da elected leader 6429d4ab4d8a33da at term 2
2016-06-30 17:32:26.327404 I | etcdserver: published {Name:dns ClientURLs:[http://localhost:9090]} to cluster d3c354cfe77b0b70
2016-06-30 17:32:26.327430 I | etcdserver: setting up the initial cluster version to 2.3
2016-06-30 17:32:26.327476 N | membership: set the initial cluster version to 2.3
2016-06-30 17:32:26.820734 I | raft: 37807cb0bf7500f6 is starting a new election at term 1
2016-06-30 17:32:26.820793 I | raft: 37807cb0bf7500f6 became candidate at term 2
2016-06-30 17:32:26.820798 I | raft: 37807cb0bf7500f6 received vote from 37807cb0bf7500f6 at term 2
2016-06-30 17:32:26.820813 I | raft: 37807cb0bf7500f6 became leader at term 2
2016-06-30 17:32:26.820819 I | raft: raft.node: 37807cb0bf7500f6 elected leader 37807cb0bf7500f6 at term 2
2016-06-30 17:32:26.820908 I | etcdserver: setting up the initial cluster version to 2.3
2016-06-30 17:32:26.821034 N | membership: set the initial cluster version to 2.3
2016-06-30 17:32:26.821081 I | etcdserver: published {Name:kubeetcd ClientURLs:[http://localhost:2379]} to cluster 2c833ae9c7555b5e
W0630 17:32:26.824396    2996 controller.go:307] Resetting endpoints for master service "kubernetes" to kind:"" apiVersion:"" 
[restful] 2016/06/30 17:32:26 log.go:30: [restful/swagger] listing is available at https://10.0.2.15:443/swaggerapi/
[restful] 2016/06/30 17:32:26 log.go:30: [restful/swagger] https://10.0.2.15:443/swaggerui/ is mapped to folder /swagger-ui/
I0630 17:32:26.857997    2996 genericapiserver.go:690] Serving securely on 0.0.0.0:443
I0630 17:32:26.858008    2996 genericapiserver.go:734] Serving insecurely on 127.0.0.1:8080
I0630 17:32:26.977240    2996 controllermanager.go:326] Starting extensions/v1beta1 apis
I0630 17:32:26.977252    2996 controllermanager.go:328] Starting horizontal pod controller.
I0630 17:32:26.977348    2996 controllermanager.go:343] Starting daemon set controller
I0630 17:32:26.977461    2996 controllermanager.go:350] Starting job controller
I0630 17:32:26.977575    2996 controllermanager.go:357] Starting deployment controller
I0630 17:32:26.977683    2996 controllermanager.go:364] Starting ReplicaSet controller
proto: no encoder for TypeMeta unversioned.TypeMeta [GetProperties]
I0630 17:32:26.977798    2996 controllermanager.go:373] Attempting to start petset, full resource map map[v1:TypeMeta:<kind:"" apiVersion:"" > groupVersion:"v1" resources:<name:"bindings" namespaced:true kind:"Binding" > resources:<name:"componentstatuses" namespaced:false kind:"ComponentStatus" > resources:<name:"configmaps" namespaced:true kind:"ConfigMap" > resources:<name:"endpoints" namespaced:true kind:"Endpoints" > resources:<name:"events" namespaced:true kind:"Event" > resources:<name:"limitranges" namespaced:true kind:"LimitRange" > resources:<name:"namespaces" namespaced:false kind:"Namespace" > resources:<name:"namespaces/finalize" namespaced:false kind:"Namespace" > resources:<name:"namespaces/status" namespaced:false kind:"Namespace" > resources:<name:"nodes" namespaced:false kind:"Node" > resources:<name:"nodes/proxy" namespaced:false kind:"Node" > resources:<name:"nodes/status" namespaced:false kind:"Node" > resources:<name:"persistentvolumeclaims" namespaced:true kind:"PersistentVolumeClaim" > resources:<name:"persistentvolumeclaims/status" namespaced:true kind:"PersistentVolumeClaim" > resources:<name:"persistentvolumes" namespaced:false kind:"PersistentVolume" > resources:<name:"persistentvolumes/status" namespaced:false kind:"PersistentVolume" > resources:<name:"pods" namespaced:true kind:"Pod" > resources:<name:"pods/attach" namespaced:true kind:"Pod" > resources:<name:"pods/binding" namespaced:true kind:"Binding" > resources:<name:"pods/exec" namespaced:true kind:"Pod" > resources:<name:"pods/log" namespaced:true kind:"Pod" > resources:<name:"pods/portforward" namespaced:true kind:"Pod" > resources:<name:"pods/proxy" namespaced:true kind:"Pod" > resources:<name:"pods/status" namespaced:true kind:"Pod" > resources:<name:"podtemplates" namespaced:true kind:"PodTemplate" > resources:<name:"replicationcontrollers" namespaced:true kind:"ReplicationController" > resources:<name:"replicationcontrollers/scale" namespaced:true kind:"Scale" > resources:<name:"replicationcontrollers/status" namespaced:true kind:"ReplicationController" > resources:<name:"resourcequotas" namespaced:true kind:"ResourceQuota" > resources:<name:"resourcequotas/status" namespaced:true kind:"ResourceQuota" > resources:<name:"secrets" namespaced:true kind:"Secret" > resources:<name:"serviceaccounts" namespaced:true kind:"ServiceAccount" > resources:<name:"services" namespaced:true kind:"Service" > resources:<name:"services/proxy" namespaced:true kind:"Service" > resources:<name:"services/status" namespaced:true kind:"Service" >  apps/v1alpha1:TypeMeta:<kind:"" apiVersion:"" > groupVersion:"apps/v1alpha1" resources:<name:"petsets" namespaced:true kind:"PetSet" > resources:<name:"petsets/status" namespaced:true kind:"PetSet" >  autoscaling/v1:TypeMeta:<kind:"" apiVersion:"" > groupVersion:"autoscaling/v1" resources:<name:"horizontalpodautoscalers" namespaced:true kind:"HorizontalPodAutoscaler" > resources:<name:"horizontalpodautoscalers/status" namespaced:true kind:"HorizontalPodAutoscaler" >  batch/v1:TypeMeta:<kind:"" apiVersion:"" > groupVersion:"batch/v1" resources:<name:"jobs" namespaced:true kind:"Job" > resources:<name:"jobs/status" namespaced:true kind:"Job" >  batch/v2alpha1:TypeMeta:<kind:"" apiVersion:"" > groupVersion:"batch/v2alpha1"  extensions/v1beta1:TypeMeta:<kind:"" apiVersion:"" > groupVersion:"extensions/v1beta1" resources:<name:"daemonsets" namespaced:true kind:"DaemonSet" > resources:<name:"daemonsets/status" namespaced:true kind:"DaemonSet" > resources:<name:"deployments" namespaced:true kind:"Deployment" > resources:<name:"deployments/rollback" namespaced:true kind:"DeploymentRollback" > resources:<name:"deployments/scale" namespaced:true kind:"Scale" > resources:<name:"deployments/status" namespaced:true kind:"Deployment" > resources:<name:"horizontalpodautoscalers" namespaced:true kind:"HorizontalPodAutoscaler" > resources:<name:"horizontalpodautoscalers/status" namespaced:true kind:"HorizontalPodAutoscaler" > resources:<name:"ingresses" namespaced:true kind:"Ingress" > resources:<name:"ingresses/status" namespaced:true kind:"Ingress" > resources:<name:"jobs" namespaced:true kind:"Job" > resources:<name:"jobs/status" namespaced:true kind:"Job" > resources:<name:"replicasets" namespaced:true kind:"ReplicaSet" > resources:<name:"replicasets/scale" namespaced:true kind:"Scale" > resources:<name:"replicasets/status" namespaced:true kind:"ReplicaSet" > resources:<name:"replicationcontrollers" namespaced:true kind:"ReplicationControllerDummy" > resources:<name:"replicationcontrollers/scale" namespaced:true kind:"Scale" > resources:<name:"thirdpartyresources" namespaced:false kind:"ThirdPartyResource" >  policy/v1alpha1:TypeMeta:<kind:"" apiVersion:"" > groupVersion:"policy/v1alpha1" resources:<name:"poddisruptionbudgets" namespaced:true kind:"PodDisruptionBudget" > resources:<name:"poddisruptionbudgets/status" namespaced:true kind:"PodDisruptionBudget" >  rbac.authorization.k8s.io/v1alpha1:TypeMeta:<kind:"" apiVersion:"" > groupVersion:"rbac.authorization.k8s.io/v1alpha1" resources:<name:"clusterrolebindings" namespaced:false kind:"ClusterRoleBinding" > resources:<name:"clusterroles" namespaced:false kind:"ClusterRole" > resources:<name:"rolebindings" namespaced:true kind:"RoleBinding" > resources:<name:"roles" namespaced:true kind:"Role" > ]
I0630 17:32:26.978038    2996 controllermanager.go:375] Starting apps/v1alpha1 apis
I0630 17:32:26.978044    2996 controllermanager.go:377] Starting PetSet controller
I0630 17:32:26.978374    2996 horizontal.go:127] Starting HPA Controller
I0630 17:32:26.978657    2996 controller.go:237] Starting Daemon Sets controller manager
I0630 17:32:26.978755    2996 pet_set.go:144] Starting petset controller
E0630 17:32:26.989760    2996 controller_base.go:144] PersistentVolumeController can't initialize caches, expected list of volumes, got: &{TypeMeta:{Kind: APIVersion:} ListMeta:{SelfLink:/api/v1/persistentvolumes ResourceVersion:10} Items:[]}
E0630 17:32:26.990580    2996 util.go:45] Metric for serviceaccount_controller already registered
I0630 17:32:26.991357    2996 attach_detach_controller.go:177] Starting Attach Detach Controller
W0630 17:32:26.992074    2996 request.go:347] Field selector: v1 - serviceaccounts - metadata.name - default: need to check if this is versioned correctly.
W0630 17:32:27.000361    2996 request.go:347] Field selector: v1 - serviceaccounts - metadata.name - default: need to check if this is versioned correctly.
I0630 17:32:27.022141    2996 kubelet.go:1194] Successfully registered node 127.0.0.1
I0630 17:32:27.070949    2996 kube2sky.go:553] Using http://127.0.0.1:8080 for kubernetes master
I0630 17:32:27.070960    2996 kube2sky.go:554] Using kubernetes API v1
I0630 17:32:27.071062    2996 kube2sky.go:622] Waiting for service: default/kubernetes
I0630 17:32:27.072441    2996 kube2sky.go:667] Successfully added DNS record for Kubernetes service.
I0630 17:32:28.011817    2996 dns.go:189] DNS started succesfully.
W0630 17:32:30.989638    2996 nodecontroller.go:787] Missing timestamp for Node 127.0.0.1. Assuming now as a timestamp.
I0630 17:32:30.989752    2996 event.go:216] Event(api.ObjectReference{Kind:"Node", Namespace:"", Name:"127.0.0.1", UID:"127.0.0.1", APIVersion:"", ResourceVersion:"", FieldPath:""}): type: 'Normal' reason: 'RegisteredNode' Node 127.0.0.1 event: Registered Node 127.0.0.1 in NodeController
I0630 17:32:31.436266    2996 reconciler.go:179] VerifyControllerAttachedVolume operation started for volume "kubernetes.io/host-path//etc/kubernetes/" (spec.Name: "addons") pod "48abed82af93bb0b941173334110923f" (UID: "48abed82af93bb0b941173334110923f")
I0630 17:32:31.536710    2996 reconciler.go:253] MountVolume operation started for volume "kubernetes.io/host-path//etc/kubernetes/" (spec.Name: "addons") to pod "48abed82af93bb0b941173334110923f" (UID: "48abed82af93bb0b941173334110923f"). 
I0630 17:32:31.536755    2996 operation_executor.go:673] MountVolume.SetUp succeeded for volume "kubernetes.io/host-path//etc/kubernetes/" (spec.Name: "addons") pod "48abed82af93bb0b941173334110923f" (UID: "48abed82af93bb0b941173334110923f").
I0630 17:32:47.996986    2996 event.go:216] Event(api.ObjectReference{Kind:"ReplicationController", Namespace:"kube-system", Name:"kubernetes-dashboard", UID:"a992decc-3ee8-11e6-b36b-16dae42c1ecf", APIVersion:"v1", ResourceVersion:"44", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' Created pod: kubernetes-dashboard-47id0
I0630 17:32:48.006740    2996 event.go:216] Event(api.ObjectReference{Kind:"Pod", Namespace:"kube-system", Name:"kubernetes-dashboard-47id0", UID:"a993fa95-3ee8-11e6-b36b-16dae42c1ecf", APIVersion:"v1", ResourceVersion:"45", FieldPath:""}): type: 'Normal' reason: 'Scheduled' Successfully assigned kubernetes-dashboard-47id0 to 127.0.0.1
I0630 17:32:48.080253    2996 reconciler.go:179] VerifyControllerAttachedVolume operation started for volume "kubernetes.io/secret/default-token-ojok4" (spec.Name: "default-token-ojok4") pod "a993fa95-3ee8-11e6-b36b-16dae42c1ecf" (UID: "a993fa95-3ee8-11e6-b36b-16dae42c1ecf")
I0630 17:32:48.180600    2996 reconciler.go:253] MountVolume operation started for volume "kubernetes.io/secret/default-token-ojok4" (spec.Name: "default-token-ojok4") to pod "a993fa95-3ee8-11e6-b36b-16dae42c1ecf" (UID: "a993fa95-3ee8-11e6-b36b-16dae42c1ecf"). 
I0630 17:32:48.183360    2996 operation_executor.go:673] MountVolume.SetUp succeeded for volume "kubernetes.io/secret/default-token-ojok4" (spec.Name: "default-token-ojok4") pod "a993fa95-3ee8-11e6-b36b-16dae42c1ecf" (UID: "a993fa95-3ee8-11e6-b36b-16dae42c1ecf").
I0630 17:32:52.144878    2996 reconciler.go:253] MountVolume operation started for volume "kubernetes.io/secret/default-token-ojok4" (spec.Name: "default-token-ojok4") to pod "a993fa95-3ee8-11e6-b36b-16dae42c1ecf" (UID: "a993fa95-3ee8-11e6-b36b-16dae42c1ecf"). Volume is already mounted to pod, but remount was requested.
I0630 17:32:52.146372    2996 operation_executor.go:673] MountVolume.SetUp succeeded for volume "kubernetes.io/secret/default-token-ojok4" (spec.Name: "default-token-ojok4") pod "a993fa95-3ee8-11e6-b36b-16dae42c1ecf" (UID: "a993fa95-3ee8-11e6-b36b-16dae42c1ecf").
I0630 17:32:53.147971    2996 reconciler.go:253] MountVolume operation started for volume "kubernetes.io/secret/default-token-ojok4" (spec.Name: "default-token-ojok4") to pod "a993fa95-3ee8-11e6-b36b-16dae42c1ecf" (UID: "a993fa95-3ee8-11e6-b36b-16dae42c1ecf"). Volume is already mounted to pod, but remount was requested.
I0630 17:32:53.149004    2996 operation_executor.go:673] MountVolume.SetUp succeeded for volume "kubernetes.io/secret/default-token-ojok4" (spec.Name: "default-token-ojok4") pod "a993fa95-3ee8-11e6-b36b-16dae42c1ecf" (UID: "a993fa95-3ee8-11e6-b36b-16dae42c1ecf").

dlorenc commented:
LGTM.
